pkgs/apps/turbomole/default.nix

```diff
@@ -30,31 +45,40 @@ in stdenv.mkDerivation rec {
   ];
 
   src = requireFile {
-    sha256 = "b903ffb4fb9d1a1dbca81187f039df99c9fc22a4eefce5566db852da4991795d";
-    name = "turbolinux${lib.replaceStrings ["."] [""] version}_TMG.tar.gz";
+    sha256 = "21518913591e7ab58b3af1c126fdc37fa31cb5507d523b4b04fbe57f63916b5e";
+    name = "turbolinux${lib.replaceStrings ["."] [""] version}-TMG.bin";
```

Comment:
The download is offered as .zip file not directly as the .bin. I think we should ingest the zip file here.